osquery/core/sqlite_util_tests.cpp

```diff
@@ -48,9 +48,53 @@ TEST_F(SQLiteUtilTests, test_aggregate_query) {
   sqlite3_close(db);
   EXPECT_EQ(err, 0);
 }
+
+TEST_F(SQLiteUtilTests, test_get_query_columns) {
+  std::unique_ptr<sqlite3, decltype(sqlite3_close)*>
+    db_managed(createDB(), sqlite3_close);
+  sqlite3* db = db_managed.get();
+
+  std::string query;
+  Status status;
+  tables::TableColumns results;
+
+  query = "SELECT * FROM time JOIN osquery_info";
```

Comment:
I would have your query explicitly ask for the columns here instead of using a *. If we add more columns to time and/or osquery_info, wouldn't the array indexing that you're doing below break?